packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec

```diff
@@ -5,10 +5,10 @@
 
 %global gem_name foreman_rh_cloud
 %global plugin_name rh_cloud
-%global foreman_min_version 2.3
+%global foreman_min_version 2.5
```

Comment:
should we set the min version to `2.4` for easier CPs in the future?

Reply:
I think it would be better to set it to 2.5. The 2.4 could use 3.y.z versions.